app/models/master_file.rb

```diff
@@ -105,6 +105,7 @@ class MasterFile < ActiveFedora::Base
   def update_media_object!
     yield
     return if mediaobject.nil?
+    mediaobject.reload
     mediaobject.set_duration!
```

Comment:
Ugh. That callback was supposed to go away completely. I missed it in my commit.